### ISSUE TITLE
Accessibility settings are cut off with 3 line "buttons"

### ISSUE THREAD

**Issue by nickvergessen** [nextcloud/server#34079]
![Bildschirmfoto vom 2022-09-14 15-42-32](https://user-images.githubusercontent.com/213943/190170596-0fdaeae9-9314-42fd-b991-e615ddd54f28.png)

**Comment by szaimen** [comment]
It broke with https://github.com/nextcloud/nextcloud-vue/pull/3192

**Comment by szaimen** [comment]
cc @CarlSchwan

**Comment by nickvergessen** [comment]
Then a `:limit-width="false"` should be enough?

**Comment by szaimen** [comment]
Yeah propbably...

**Comment by CarlSchwan** [comment]
Yeah that should be enough

**Comment by szaimen** [comment]
Are other settings panels broken, too? Then we should probably fix them in one go if possible... (I suppose the activity/notification settings are borken as well?)

**Comment by nickvergessen** [comment]
They live in the respective apps, so bumping the vue lib there will raise the problem and should then extend the section properties if needed
